cpp/src/Ice/HttpParser.cpp

```diff
@@ -672,10 +672,10 @@ IceInternal::HttpParser::getHeader(const string& name, string& value, bool toLow
     return false;
 }
 
-map<string, string>
+map<string, string, std::less<>>
```

Comment:
Should we use `HeaderDict`  from Connection.h?

Reply:
I didn't because `HttpParser.{h,cpp}` barely includes any Ice code, only Ice/Config.h and Ice/StringUtil.h in the .cpp file.